src/environmentd/tests/auth.rs

```diff
@@ -1002,6 +1004,7 @@ fn test_auth_base() {
                 auth: &WebSocketAuth::Basic {
                     user: "bad user".to_string(),
                     password: frontegg_password.to_string(),
+                    options: Some(BTreeMap::default()),
```

Comment:
Why are these None sometimes or Some but empty sometimes? Does the Auth struct even need to be an option? If it's a btreemap but not specified in the json, what does serde do?

Reply:
Originally I did this just to make sure we had coverage of both cases, but now that options is not an Option, I changed all of these to `BTreeMap::default`